### PR TITLE
Added RequireAuth/Fixed minor issues

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Added RequireAuth
Fixed minor issues

### CODE REVIEW

File: src/Pages/VideoPage/VideoPage.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
 .video-info-container {
   display: flex;
   justify-content: flex-start;
-  margin-left: 13.3rem;
+  margin-left: 15.2rem;
   border: lightgray;
   background-color: var(--drop-hover-color);
   width: 1024px;
```
Comment (ashishkatiyar2404) [inline src/Pages/VideoPage/VideoPage.css:16]: use rem or % if more thn 10px

File: src/Components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
 .sidebar-links-name {
   margin-left: 0.8rem;
   text-decoration: none;
-  font-size: 14px;
+  font-size: 1rem;
   white-space: nowrap;
+  color: var(--component-black-color);
+}
+.sidebar-icon {
+  color: #606060;
```
Comment (ashishkatiyar2404) [inline src/Components/Sidebar/Sidebar.css:21]: use color variables

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested some changes u can look into it.
The file structure is also good.
You can merge now.

**Comment by ShrutiLakhani** [comment]
Thanks for the review. Merging to Dev

**Pull request merged**